XenCenterLib/EncryptionUtils.cs

```diff
@@ -66,7 +66,7 @@ public static string Protect(string data)
         {
             byte[] dataBytes = Encoding.Unicode.GetBytes(data);
 
-            byte[] protectedBytes = ProtectedData.Protect(dataBytes, GetSalt(), DataProtectionScope.CurrentUser);
+            byte[] protectedBytes = ProtectedData.Protect(dataBytes, GetSalt(), DataProtectionScope.LocalMachine);

```

Comment:
This would change how XenCenter stores data.
While I see why this change was made, this would require users to know that they will need to trust not only administrators, but all the local users on a machine they are using.